cms/djangoapps/modulestore_migrator/rest_api/v1/views.py

```diff
@@ -0,0 +1,140 @@
+"""
+API v1 views.
+"""
+import logging
+
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from rest_framework.permissions import IsAdminUser
+from rest_framework.response import Response
+from rest_framework import status
+from user_tasks.models import UserTaskStatus
+from user_tasks.views import StatusViewSet
+
+from cms.djangoapps.modulestore_migrator.api import start_migration_to_library
+from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
+
+from .serializers import ModulestoreMigrationSerializer, StatusWithModulestoreMigrationSerializer
+
+
+log = logging.getLogger(__name__)
+
+
+class ImportViewSet(StatusViewSet):
```

Comment:
```suggestion
class MigrationViewSet(StatusViewSet):
```
Although this will be used in the "Import Course to Library" user interface, let's stick with "migration" as the term we use in the API and the backend.